overlord/snapstate/snapmgr.go

```diff
@@ -149,6 +149,10 @@ type SnapSetup struct {
 	// DownloadBlobDir is the directory where the snap blob is downloaded to. If
 	// empty, dir.SnapBlobDir is used.
 	DownloadBlobDir string `json:"download-blob-dir,omitempty"`
+
+	// Registries is the set of registries that the snap plugs, identified by
+	// account and registry name pairs.
+	Registries [][2]string
```

Comment:
Would a small struct with named fields be better here? I don't have a super strong opinion, but I think that'd be maybe more flexible for adding fields, and also more readable for consumers?

Reply:
Changed to a struct for better readability